### ISSUE TITLE
Trait Suggestion: Novak space S109 is totally disconnected P47

### ISSUE THREAD

**Issue by GeoffreySangston** [pi-base/data#1055]
## Trait Suggestion

The space Novak space [S109](https://topology.pi-base.org/spaces/S000109) is totally disconnected P47, but this fact is not known to pi-Base today: [link to pi-Base](https://topology.pi-base.org/spaces/S000109/properties/P000047).

## Proof/References

This [MSE thread](https://math.stackexchange.com/questions/3411402/stone-cech-compactification-of-discrete-space-is-totally-disconnected) says that the Stone-Cech compactification of a discrete space is totally disconnected; there's also a Stacks project [page](https://stacks.math.columbia.edu/tag/090C#:~:text=In%20particular%2C%20we%20conclude%20that,2). Since 'totally disconnected' is a hereditary property, and $X \subset \beta\omega$ by definition, it follows that $X$ is totally disconnected.

## Remarks

Someone who actually understands Novak's space (I just know its a subspace of $\beta \omega$) might know if it's totally separated or extremally disconnected, either of which would be an improvement.

**Comment by Moniker1998** [comment]
I would say to not aim for proving traits that could be resolved in a more efficient manner.

**Comment by GeoffreySangston** [comment]
@Moniker1998 What's the more efficient manner? I raised the issue so we could discuss it.

**Comment by Moniker1998** [comment]
I'm saying this because we are already piling up in terms of raised issues (over 80 open ones). So lets try to be efficient.
I've mentioned this just to make sure that you are aware of this. I would try to avoid easier to prove traits for the ones that are actually less obvious, unless a trait couldn't be proven from other ones. 

In this case we can say the space is extremally disconnected, since its a dense subspace of extremally disconnected space.

**Comment by GeoffreySangston** [comment]
@Moniker1998 I disagree with the sentiment. Novak's space sat around for a long time without pi-base knowing if it is contractible or not. By raising the issue, now you point out it is extremally disconnected. Had I not raised the issue, probably pi-base would not have known if it was contractible or not for quite a long time.

**Comment by Moniker1998** [comment]
I don't think that raising an issue for addition of a weaker property is a good place to determine if we can prove a stronger property or not. 
Perhaps there should be a separate place for such questions, maybe in discussions. @StevenClontz

**Comment by GeoffreySangston** [comment]
@Moniker1998 I don't think that's a fair representation of this thread. A fair representation of this thread is twofold

1. To share information not known to pi-base (which did not even know if this space is not contractible). My long term goal is to fill out all of the information related to contractible + simply connected + a few other properties yet to be added. 
2. To allow the community an opportunity to refine that information. In my view, it's not a problem if a trait gets added to pi-base which is eventually removed, if adding the trait in the first place did solve a problem and was not very difficult.

**Comment by Moniker1998** [comment]
> To allow the community an opportunity to refine that information. In my view, it's not a problem if a trait gets added to pi-base which is eventually removed, if adding the trait in the first place did solve a problem and was not very difficult.

I do see this as a slight problem if it would keep to consistently occur. It creates unnecessary clutter, and as I said the issues keep on piling up as well.

**Comment by GeoffreySangston** [comment]
@Moniker1998 Do you have git PR privileges? I'm sure everyone would welcome your git contributions if not. That seems like a secondary issue @StevenClontz or @prabau might be able to address.

**Comment by Moniker1998** [comment]
> I think if the position you advocate is adopted, then it gate keeps contribution to the website to only those members with strong knowledge of all 200 properties in pi-base. That seems like a clear bad outcome.

I don't see how it is so. ideologically I am advocating for a more throughout approach towards issues, more research of a space and traits involved, current theorems on pi-base.  
Practically, I am suggesting for there to be a place to ask if a stronger property can be proven, outside of 'issues' tab.

**Comment by GeoffreySangston** [comment]
> > To allow the community an opportunity to refine that information. In my view, it's not a problem if a trait gets added to pi-base which is eventually removed, if adding the trait in the first place did solve a problem and was not very difficult.
> 
> I do see this as a slight problem if it would keep to consistently occur. It creates unnecessary clutter, and as I said the issues keep on piling up as well.

Just wanted to add that I agree with you that this could lead to clutter. I think it's a matter of finding a balance though. Also, a few times now I've opened PR's to remove redundant traits from spaces, so I hope people are aware that I am aware of the potential issue.

**Comment by StevenClontz** [comment]
There's some good discussion here, and I think that as we (hopefully!) continue to grow, we need to have these discussions on community norms and expectations. Everything we've settled on during community calls is (or should be) in the https://github.com/pi-base/data/wiki

I'm always a fan of not letting the perfect be the enemy of the good. Furthermore, the more picky we are, the more friction we create for new community members to get involved.

So if an issue is opened that can be improved, I think we should focus on constructive feedback (e.g. suggest changing totally disconnected to extremally disconnected, or suggest that a theorem can be added instead of a single trait), and try to efficiently refine the issue to something that should be implemented, or determine that the issue can be closed or merged with another issue.

**Comment by prabau** [comment]
@StevenClontz @GeoffreySangston @Moniker1998 
I would like to suggest an additional thing.  The number of **issues**, especially for suggested traits, is becoming out of hand.

Some of these trait issues require a discussion, but probably most of them don't.  For those that don't, it would be more efficient for the proposer to bypass writing an issue and directly write a PR instead.

That would serve two purposes: (1) more efficient overall process by going more quickly to implementation; (2) limit the rate at which the proposer generates issues/PRs.  I.e., they would (or maybe should) wait until some of these PRs are merged before create new ones.  I.e., should there be some form of (not enforced, but recommended) limit on the number of PRs currently pending from anyone?  (Not even sure I agree about that one, but just writing this down for discussion.)

And like Steven, I don't really see a problem with adding a trait that later gets superseded by a stronger one or a theorem.

Anyway, these are possibilities to reduce buraucracy and improve overall efficiency.

Could be discussed in a community meeting.

**Comment by StevenClontz** [comment]
For those of us comfortable writing PRs, I definitely think that filling in trait gaps is best done directly. I'd rather have a dozen small one-file trait PRs than a dozen issues suggesting them. I'll update the issue template to suggest this.

**Comment by prabau** [comment]
@StevenClontz I had a few additional small suggestions for improving the new issue templates.  Should I write an issue for discussion?

**Comment by StevenClontz** [comment]
> @StevenClontz I had a few additional small suggestions for improving the new issue templates. Should I write an issue for discussion?

Find with me, or propose a PR directly if you think you have a solid improvement.
